include/igl/triangle/triangulate.cpp

```diff
@@ -18,15 +18,17 @@
 #  define IGL_PREVIOUSLY_DEFINED_VOID VOID
 #  undef VOID
 #endif
-#define ANSI_DECLARATORS
+#define ANSI_DECLARATORS 1
 #define REAL double
 #define VOID int
+#define TRILIBRARY 1
```

Comment:
Same question as above. Where is this define used?

Reply:
Again, this is used inside Triangle. Without TRILIBRARY defined, the name of the main function is "main" instead of "triangulate". I think libigl defines this elsewhere when it's set to build with Triangle, but it didn't work unless this was added in when making Triangle header-only.